cuqi/experimental/mcmc/__init__.py

```diff
@@ -1,4 +1,106 @@
-""" Re-implementation of sampler module in a more object oriented way. """
+"""
+Re-implementation of sampler module in a more object-oriented way.
+
+Main changes for users
+----------------------
+
+1. Sampling API
+   ^^^^^^^^^^^^
+
+   Previously one would call the `.sample` or `sample_adapt` methods of a sampler instance to sample from a target distribution and store the samples as the output as follows:
+
+   .. code-block:: python
+
+      from cuqi.sampler import MH
+      from cuqi.distribution import DistributionGallery
+
+      # Target distribution
+      target = DistributionGallery("donut")
+
+      # Set up sampler
+      sampler = MH(target)
+
+      # Sample from the target distribution
+      samples = sampler.sample(Ns=100, Nb=100)
```

Comment:
Below burnthin is used. To align with this, either use also burnthin here, or explain how the same is achived with Nb=100 (no thinning I suppose).

Reply:
I updated the description to explain.